server/utils/ssr.tsx

```diff
@@ -98,19 +101,19 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 			<meta
 				key="t2"
 				property="og:title"
-				content={useCollectionTitle ? collection.title : title}
+				content={useCollectionTitle ? collection.title ?? undefined : title}
```

Comment:
`content` cannot be null, so a lot of these silly things.
Again, maybe `collection.title` should just be `string`